config.py

```diff
@@ -8,6 +8,12 @@ class Config(object):
     LOGGING_LEVEL = os.getenv('LOGGING_LEVEL', 'INFO')
 
     BACKSTAGE_API_URL = os.getenv('BACKSTAGE_API_URL', 'http://localhost:8001/backstage-api/v1')
+    RESPONSE_OPERATIONS_UI_SECRET = os.getenv('RESPONSE_OPERATIONS_UI_SECRET', "secret")
+
+    PASSWORD_MATCH_ERROR_TEXT = 'Your passwords do not match'
```

Comment:
I think this text should be kept somewhere else, it's not really config

Reply:
It's not used for now, so I'll remove until we get the rad designs